src/components/ShellView/ResultGraph.vue

```diff
@@ -190,6 +190,38 @@
         </div>
       </div>
     </div>
+    <!-- TOOLTIP HOVER -->
+    <div
+      v-show="tooltipVisible"
+      class="result-graph__graph-tooltip"
+      :style="{ left: tooltipX + 'px', top: tooltipY + 'px' }"
+    >
+      <div class="result-graph__tooltip-header">
+        <span
+          class="badge bg-primary"
+          :style="{
+            backgroundColor: `${getColor(hoveredLabel)} !important`,
+            color: `${getTextColor(hoveredLabel)} !important`,
+          }"
+        >
+          {{ hoveredLabel }}</span>
+      </div>
+      <p>{{ hoveredIsNode ? 'Node' : 'Rel' }}</p>
+      <hr>
+      <table class="table table-sm table-borderless">
+        <tbody>
+          <tr
+            v-for="property in hoveredProperties"
+            :key="property.name"
+          >
+            <th scope="row">
+              {{ property.name }}
+            </th>
+            <td>{{ property.value }}</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
```

Comment:
This should be refactored to its own component. It does not have a lot of HTML code, but has a lot of CSS code.